sat/cli/bootsys/util.py

```diff
@@ -229,3 +229,19 @@ def get_ssh_client(host_keys=None):
     ssh_client.set_missing_host_key_policy(WarningPolicy)
 
     return ssh_client
+
+
+def hard_power_off(prompt):
```

Comment:
I think you can use `pester_choices` or `prompt_continue` instead of this function, but nice job writing a docstring for it.

If you do end up needing the function, I'd suggest naming it so it doesn't sound like it does a hard power off. It really just prompts if one should be performed, so a name such as `hard_power_off_requested` or `prompt_for_hard_power_off` may be better. The former sounds better in an `if` statement, while the former is more descriptive about what the function does.